README.rst

```diff
@@ -63,13 +65,27 @@ Settings list
 -------------
 All settings are contained in a ``PUSH_NOTIFICATIONS_SETTINGS`` dict.
 
-In order to use GCM, you are required to include ``GCM_API_KEY``.
-For APNS, you are required to include ``APNS_CERTIFICATE``.
+In order to use GCM, you are required to include one of ``GCM_API_KEY``, ``GCM_API_KEYS``, or ``GCM_API_KEYS_MODEL``.
+For APNS, you are required to include ``APNS_CERTIFICATE``, ``APNS_CERTIFICATES``, or ``APNS_CERTIFICATES_MODEL``.
 
 - ``APNS_CERTIFICATE``: Absolute path to your APNS certificate file. Certificates with passphrases are not supported.
+- ``APNS_CERTIFICATES``: A dictionary reflecting separate application IDs to separate APNS certificate files.
+- ``APNS_CERTIFICATES_MODEL``: A dictionary containing description of a database model to reflect
```

Comment:
really these options was added trying to solve two different problems, backward compatibility on the one hand, and difference between usecases on the other one.

There was 3 noticeable different usecases found in discursions around this PR before.

The first one is old-style using for such users who won't change anything and happy to use one push notification applivation per every platform.

The second one is for multiple but static set of applications per platform. Such users may be also aware about potential security risks of using django models to store private keys, so such static option is an ideal for their requirements.

The third one is for users like me and you. We need to have dynamic set of applications managed over the web by web-users, so we need to store private keys in a database.

And yes, you right, we need detail explanation and even ready-to-use examples and programming templates in perspective of package grow in this direction.